### PR TITLE
Terraform 0.5.0

### PR DESCRIPTION
**Pull request by number5**
- bumped the version and redo all go_deps
- fixed hash with fat arrow and extra white space

re: #39516

### CODE REVIEW

File: Library/Formula/terraform.rb [hunk 1]
```diff
@@ -3,188 +3,85 @@
...
+github.com/rackspace/gophercloud
+github.com/satori/go.uuid
+github.com/soniah/dnsmadeeasy
+github.com/vaughan0/go-ini
+github.com/xanzy/go-cloudstack
+  ]
+
+  terraform_deps.each do |x|
+    go_resource x do
+      url "https://#{x}.git"
```
Comment (dunn) [inline Library/Formula/terraform.rb]: Go resources need a revision to make sure the formula doesn't break because of a dependency.

**Comment by jacknagel** [comment]
ping

**Comment by number5** [comment]
sorry busy right now, will update the pull request tonight.

**Comment by number5** [comment]
Updated as per @dunn suggested, passed merged build, please review, thanks.

**Comment by number5** [comment]
ping #39516

**Pull request closed without merging**